runtime/acala/src/lib.rs

```diff
@@ -1268,18 +1268,37 @@ impl pallet_proxy::Config for Runtime {
 parameter_types! {
 	pub const ChainId: u64 = 787;
 	pub const NewContractExtraBytes: u32 = 10_000;
-	pub StorageDepositPerByte: Balance = deposit(0, 1);
 	pub NetworkContractSource: H160 = H160::from_low_u64_be(0);
 	pub DeveloperDeposit: Balance = 100 * dollar(ACA);
 	pub DeploymentFee: Balance = 10000 * dollar(ACA);
 }
 
+#[derive(Clone, Encode, Decode, PartialEq, Eq, RuntimeDebug, TypeInfo)]
+pub struct StorageDepositPerByte;
+impl<I: From<Balance>> frame_support::traits::Get<I> for StorageDepositPerByte {
+	fn get() -> I {
+		// NOTE: use 18 decimals
+		I::from(100 * dollar(ACA))
```

Comment:
https://github.com/AcalaNetwork/Acala/issues/1640 should provide a better way to convert decimals